jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
> When i use (in-package ~~~) in apc-interface.l, I can run (jsk_arc2017_baxter::apc-init) but *ri* was not set.
in-package shows really weird effect, and I do not want to use it.

Maybe the `jsk_arc2017_baxter::*ri*` is set.

```
eus$ make-package 'jsk_arc2017_baxter
#<package #X1054d4d40 JSK_ARC2017_BAXTER>
eus$ in-package 'jsk_arc2017_baxter
#<package #X1054d4d40 JSK_ARC2017_BAXTER>
JSK_ARC2017_BAXTER:eus$ setq *ri* 1
1
JSK_ARC2017_BAXTER:eus$ in-package 'user
#<package #X7fe4cd8156b8 USER>
eus$ *ri*
sh: *ri*: command not found
32512
eus$ jsk_arc2017_baxter::*ri*
1
```